library/templates/nycoc_checkbook.yml

```diff
@@ -0,0 +1,49 @@
+dataset:
+  name: &name nycoc_checkbook
+  version: "{{ version }}"
+  acl: private 
+  
+  source:
+    url:
+      path: .library/datasources/checkbook_citywide_agencies.csv
+      subpath: ""
+
+    geometry:
+      SRS: null
+      type: NONE
+
+    options:
+      - "AUTODETECT_TYPE=NO"
+      - "EMPTY_STRING_AS_NULL=YES"
+
+  destination: 
+    name: *name
+
+    geometry:
+      SRS: null
+      type: NONE
+
+    options: 
+      - "OVERWRITE=YES"
+      - "PRECISION=NO"
+
+    fields: []
+    sql: null 
+
+  info:
+    description: |
+      The NYCOC Checkbook dataset was created by EDM DE Fellows in response to a request from Capital Planning 
+      to explore the feasibility of creating a data product focused on collecting geospatial data to historical 
+      liquidations data to better inform future spending decisions at the community level. 
```

Comment:
just to help future us understand what `nycoc` means, might be nice to change `The NYCOC Checkbook dataset` to  `The NYC Office of the Comptroller Checkbook`